tests/e2e/conftest.py

```
@@ -27,14 +27,14 @@
 from pydantic import SecretStr
 
-from py_strapi import AsyncClient, StrapiConfig, SyncClient
+from strapi_kit import AsyncClient, StrapiConfig, SyncClient
 
 if TYPE_CHECKING:
     from collections.abc import AsyncGenerator
 
 # Default configuration
 DEFAULT_STRAPI_URL = "http://localhost:1337"
-DEFAULT_API_TOKEN = "e2e-test-token-py-strapi-12345"
+DEFAULT_API_TOKEN = "e2e-test-token-strapi-kit-12345"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "DEFAULT_API_TOKEN"

(S105)